controllers/common.go

```diff
@@ -0,0 +1,21 @@
+package controllers
+
+const (
+	KuadrantAppName = "kuadrant"
+)
+
+var (
+	AppLabelKey   = "app"
+	AppLabelValue = KuadrantAppName
+)
+
+func CommonLabels() map[string]string {
+	return map[string]string{
+		AppLabelKey:                    AppLabelValue,
+		"app.kubernetes.io/component":  KuadrantAppName,
+		"app.kubernetes.io/managed-by": "kuadrant-operator",
+		"app.kubernetes.io/instance":   KuadrantAppName,
+		"app.kubernetes.io/name":       KuadrantAppName,
+		"app.kubernetes.io/part-of":    KuadrantAppName,
+	}
+}
```

Comment:
Copied the layout and labels from https://github.com/Kuadrant/kuadrant-operator/blob/main/pkg/openshift/consoleplugin/common.go

Reply:
I've been adding a few common vars and helper funcs for things like labeling internal resources we create, etc, including one to identify the resource as "managed-by" kuadrant, here: https://github.com/Kuadrant/kuadrant-operator/blob/b6494ca686716a4fb90f03ae8360fb25a1ab7745/controllers/state_of_the_world.go#L43

(and also at the top of specific workflow files, when the var is not just as common as these ones.)

Happy to move those here once this is merged, if you believe it would help having them all in one place rather than separated by area.